cpp/perspective/src/include/perspective/exprtk.h

```diff
@@ -702,7 +733,7 @@ namespace details {
                 }
 
                 // degrees to gradians
-                rval.set(v.to_double() * (20.0 / 9.0));
+                rval.set(v.to_double() * (10.0 / 9.0));
```

Comment:
👀

Reply:
Yeah the ration for converting between degrees to grad is 1.1111...